trl/trainer/online_dpo_config.py

```diff
@@ -9,7 +9,7 @@ class OnlineDPOConfig(TrainingArguments):
     r"""
     Configuration class for the [`OnlineDPOTrainer`].
 
-    Using [`~transformers.HfArgumentParser`] we can turn this data class into
```

Comment:
While I agree it's a data class here, that phrase seems to be all over the transformers doc, so I'd favour consistency.